test/e2e/support/utils/meta-key.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Platform-specific meta key.
+ *
+ * @see pressWithModifier
+ *
+ * @type {string}
+ */
+export const META_KEY = process.platform === 'darwin' ? 'Meta' : 'Control';
```

Comment:
This looks similar to:
https://github.com/WordPress/gutenberg/blob/master/packages/keycodes/src/index.js#L47

I'm wondering if we could use https://github.com/WordPress/gutenberg/blob/master/packages/keycodes/src/index.js#L61
instead or something similar to avoid logic duplication.

I bet it would be easier to perform it in the follow-up PR. For this one, can we consolidate those modifiers in one file? I see that `ACCESS_MODIFIER_KEYS` is also in their own file. It would read easier if it was colocated with `pressWithModifier` function.

Reply:
Yes, placing this into follow-up PR is a good idea. Will use your hints when I will create that.
For now I've colocated them here:
https://github.com/WordPress/gutenberg/pull/11701/commits/90dec9f2e695de4b1c0f2ecb9aedfd682f568e96